### PR TITLE
API: Endpoint to get an organization's listings

### PR DESCRIPTION
**Pull request by twinsfan421**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

This would provide an api endpoint dev.to/api/organizations/{org_username}/listings to get a list of an organization's listings.
As discussed in issue 9212, This is the 3rd of 4 related endpoints that will include:

`dev.to/api/organizations/{org_username}/articles`

## Related Tickets & Documents
#9212 #10931 #11514 

## Added tests?

- [x] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [ ] I need help with writing tests

## Added to documentation?

- [x] [Developer Docs](https://docs.forem.com) and/or
      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/)
- [ ] README
- [ ] No documentation needed

### CODE REVIEW

**Comment by twinsfan421** [comment]
@rhymes Let me know if you want me to make any changes. I didn't include the user/organization in the jbuilder but I could include them and use the same one as the listing index jbuilder.   Thanks

**Comment by rhymes** [comment]
@twinsfan421 you read my mind. I think it should be exactly the same of https://dev.to/api/listings (including the category filter maybe?). This way we could technically recycle caching in future. Thank you for taking this on! It'll also simplify the API spec as the resource it's the same :)

**Comment by twinsfan421** [comment]
@rhymes I updated it to include the user/organization and the category filter

**Review by rhymes — COMMENTED** [review]
Thank you again @twinsfan421, I left a note about simplifying the OpenAPI spec but the overall code works very well!

File: docs/api_v0.yml [hunk 1]
```diff
@@ -740,6 +749,50 @@ components:
         organization:
           $ref: "#/components/schemas/SharedOrganization"
 
+    ListingByOrganization:
```
[the remark this answers is not in the document]
Comment (twinsfan421) [reply docs/api_v0.yml]: I just pushed up an update for this. Thanks

**Review by rhymes — APPROVED** [review]
LGTM!

Thanks @twinsfan421

**Review by benhalpern — APPROVED** [review]

**Pull request merged**